docs-src/source/modules/gen/index.rst

```diff
@@ -0,0 +1,15 @@
+gen module - Generic OTP-style calls
+====================================
+
+The gen module implements Erlang style behaviours ``gen_server``, and maybe
```

Comment:
Clarification: `gen.py` has overlapping functions with `gen.erl` in OTP, where it contains generic call/response code between `gen_*` everything

Reply:
ok, I'll update